packages/components/src/DataList/components/DataListEmptyState/DataListEmptyState.css

```diff
@@ -8,3 +9,50 @@
   align-items: center;
   flex: 1;
 }
+
+.emptyStateWrapper h3 {
+  text-align: center;
+}
+
+.fakeData {
```

Comment:
Referencing the DOM/CSS from existing empty states in Jobber. I think we can probably move what I have in the TSX out to a subcomponent or something - I don't love that I have to get into negative z-indexing here to get the fake data beneath the message and action, but I think the alternative would be adding more wrappers around those and absolutely positioning them.